src/test/integration/gecko-profile-to-display.test.js

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I decided to create a new directory because I intend these tests to not assert a lot of conditions, but rather just check that we can go through essential scenarios without problems.